editor/src/components/canvas/canvas-strategies/strategies/resize-grid-strategy.ts

```diff
@@ -117,16 +119,32 @@ export const resizeGridStrategy: CanvasStrategyFactory = (
       const calculatedValue = toFirst(valueOptic, calculatedValues.dimensions)
       const mergedValue = toFirst(valueOptic, mergedValues.dimensions)
       const mergedUnit = toFirst(unitOptic, mergedValues.dimensions)
+      const isFractional = isRight(mergedUnit) && mergedUnit.value === 'fr'
+
+      function newResizedValue(
+        current: number,
+        increment: number,
+        precision: 'coarse' | 'precise',
+      ): number {
+        const newValue = current + increment
+        if (precision === 'precise') {
+          return newValue
+        } else if (isFractional) {
```

Comment:
nit but it's a little weird that this is not a parameter of the function

Reply:
good point, done in https://github.com/concrete-utopia/utopia/pull/6104/commits/4d0371ca34e508090b1960be9afdaf63ae7652cc